photonlib-cpp-examples/apriltagExample/networktables.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Maybe remove this and put in gitignore? Not sure what best practice is but it's what we've been doing on 1736 so far